pages/games/[hash].js

```diff
@@ -1,34 +1,80 @@
-import styles from "../../styles/Game.module.css";
+import { useState } from "react";
 
 function Game({ data }) {
+  const PLAYER = "PLAYER";
+  const OPPONENT = "OPPONENT";
+
+  const [visibleBoard, setVisibleBoard] = useState(PLAYER);
+
+  function toggleBoard() {
+    if (visibleBoard == PLAYER) {
+      setVisibleBoard(OPPONENT);
+    } else {
+      setVisibleBoard(PLAYER);
+    }
```

Comment:
```suggestion
    setVisibleBoard(visibleBoard === PLAYER ? OPPONENT : PLAYER);
```

Shorter and strict equality